backend/eatandmeet/api/views.py

```diff
@@ -0,0 +1,14 @@
+from django.shortcuts import render
+from django.contrib.auth.models import User
+from rest_framework import viewsets
+from .models import Event, User
+from .serializers import EventSerializer, UserSerializer
+
+# Create your views here.
+class UserViewSet(viewsets.ModelViewSet):
+    queryset = User.objects.all()
+    serializer_class = (UserSerializer)
+
+class EventViewSet(viewsets.ModelViewSet):
```

Comment:
```suggestion

class EventViewSet(viewsets.ModelViewSet):
```

Two empty lines between classes.